tests/e2e/testcases/subnet/local/suite.go

```diff
@@ -242,14 +242,17 @@ var _ = ginkgo.Describe("[Local Subnet]", ginkgo.Ordered, func() {
 		commands.CreateSubnetEvmConfig(subnetName, utils.SubnetEvmGenesisPath)
 
 		deployOutput := commands.DeploySubnetLocally(subnetName)
+		fmt.Println(deployOutput)
 		rpcs, err := utils.ParseRPCsFromOutput(deployOutput)
 		if err != nil {
 			fmt.Println(deployOutput)
 		}
 		gomega.Expect(err).Should(gomega.BeNil())
 		gomega.Expect(rpcs).Should(gomega.HaveLen(1))
 
-		deployOutput = commands.DeploySubnetLocally(subnetName)
+		out, err := commands.DeploySubnetLocallyWithArgsAndOutput(subnetName, "", "")
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+		deployOutput = string(out)
 		rpcs, err = utils.ParseRPCsFromOutput(deployOutput)
 		if err == nil {
 			fmt.Println(deployOutput)
```

Comment:
we need e2e for teleporter

Reply:
will add it together with teleporter msg command